src/documents.ts

```diff
@@ -1,3 +1,10 @@
+export interface AlternateLanguage {
+  id: string;
+  uid: string;
```

Comment:
This should be `uid?: string`, I think (based on experimenting with the api)

Reply:
Indeed, `uid` is optional in Prismic